config/locales/gcse_details.yml

```diff
@@ -4,11 +4,11 @@ en:
       maths: Add maths GCSE grade 4 (C) or above, or equivalent
       english: Add English GCSE grade 4 (C) or above, or equivalent
       science: Add science GCSE grade 4 (C) or above, or equivalent
-  gcse_edit_details:
+  gcse_edit_grade:
     page_titles:
-      maths: Maths qualification grade and year
-      english: English qualification grade and year
-      science: Science qualification grade and year
+      maths: What grade is your maths qualification?
+      english: What grade is your english qualification?
+      science: What grade is your science qualification?
```

Comment:
Potentially:

```yaml
gcse_edit_grade:
  page_title: What grade is your %{subject} qualification?
```

Then for example you could do `t('gcse_edit_details.page_title', subject: 'Maths')`.

Reply:
This is a pro tip @chubberlisk, I remember using this on TVS but I totally forgot about this, thanks! ⭐️